app/controllers/concerns/curation_concerns/curation_concern_controller.rb

```diff
@@ -203,5 +204,20 @@ def search_result_document(search_params)
         raise CanCan::AccessDenied.new(nil, :show) if document_list.empty?
         document_list.first
       end
+
+      def authorize_workflow_role!
+        if user_cannot_access_due_to_workflow?(presenter.solr_document)
```

Comment:
Will this presenter ever find anything? Won't the search builder filter out the suppressed results?